src/lib/getPageOffset.ts

```diff
@@ -0,0 +1,5 @@
+import { ENTRIES_PER_PAGE } from '@lib/constants';
+
+export default function getPageOffset(page: number | string): number {
+	return (+page - 1) * ENTRIES_PER_PAGE;
```

Comment:
`(+'' - 1) * 10 = -10`, so I wonder if it would make sense to have this be:

```suggestion
	return Math.max(+page - 1, 0) * ENTRIES_PER_PAGE;
```

Probably not needed at this point, but a little more defensive.